### DISCUSSION TITLE
Kodi

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#335] [General]
Who knows if there are some good Kodi alternatives?
Thanks

**Comment by janply** [comment]
There are many of them - Plex, Emby,  Media Portal... Do you need free or paid options?

**Reply by hernessteve** [reply]
Stremio is very good. 
Here you can read a good article on this matter - [](https://www.firesticktricks.com/best-kodi-alternatives.html)https://www.firesticktricks.com/best-kodi-alternatives.html.
Good luck.
